tfutils/base.py

```diff
@@ -144,29 +146,29 @@ def __init__(self,
         if not os.path.isdir(self.cache_dir):
             os.makedirs(self.cache_dir)
 
-        if restore:
-            self.restore_model(force_fetch=force_fetch)
+        self.load_model(force_fetch=force_fetch)
 
         self.start_time_step = time.time()  # start timer
 
-    def restore_model(self, force_fetch=False):
+    def load_model(self, force_fetch=False):
```

Comment:
@qbilius I think the Saver should always initialize variables, instead of doing it sometimes (when restore is true) and not others. So I renamed the method "load_model", which will always initialize variables -- even if just from scratch.